src/components/AddVehicle.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  Container, Row, Col, Alert, Form, Button,
+} from 'react-bootstrap';
+import { MdCheck } from 'react-icons/md';
+import { AiOutlineRightCircle } from 'react-icons/ai';
+import { addVehicle } from '../redux/vehicles/vehicles';
+import '../styles/AddVehicle.scss';
+import { getAuth, uploadFile } from '../redux/uploadcare/uploadcare';
+
+const DEFAULT_VALUES = {
+  brand: 'Yamaha',
+  model: '2003',
+  description: 'Really good bike',
+  price: 10000,
+  image: 'image.jpg',
+  visible: true,
+};
+
+function AddVehicle() {
```

Comment:
- [ ] All over your project, you are using the 'function' keyword.  Kindly refactor these `function` declarations for ES6 syntax.


Reason:  Almost the whole software industry has moved away from the `function` keyword.  

That way, your project will comply with industry standards 💪